### DISCUSSION TITLE
Important notice about proxy discussions

### DISCUSSION THREAD

**Discussion by akabutnicer** [3kh0/ext-remover#1325] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
From now on, proxy discussions are prohibited to essentially prevent, "proxy generals". If these violations are further invoked, I shall delete the post(s) in question on my alt. It's not an opposition that these proxies be shared, but rather a matter that I alongside everyone else would like to maintain our already crumbling reputation (might have partially been my fault but we don't talk about that).

**Comment by awsome-webdev** [comment]
ya that makes since

**Comment by Luigi-Master** [comment]
So we can still post proxies but to not complain if they don't work or the post will be taken off, Is that right?

**Reply by akabutnicer** [reply]
no, not exactly. I could care less if y'all argue about links not working; I just don't want the links to be shared in general to prevent more ostracizing statements about 3kh0 to be created

**Comment by schoolexploitkid** [comment]
please do not link leak, as creators of the links have to spend money to maintain them. if u link leak and they get blocked, it is a waste of their effort and time.

**Reply by S-PScripts** [reply]
yeah, unless they're ones like vercel, netlify, maybe freedns.

basically just community made ones

**Comment by akabutnicer** [comment]
minor update:

upon checking my alt it appears my collaborator privileges to this repository have been revoked. therefore, it's the other collaborators' job to close unwanted discussions.

**Reply by schoolexploitkid** [reply]
who is ur alt anyways

**Reply by S-PScripts** [reply]
> who is ur alt anyways

@Aka-but-nice

**Reply by SlitheringLizard1** [reply]
wait what I thought akabutnicer was the alt
whatever

**Reply by S-PScripts** [reply]
> wait what I thought akabutnicer was the alt whatever

yeah i thought so too

**Reply by akabutnicer** [reply]
> wait what I thought akabutnicer was the alt whatever

basically my original account was @Aka-but-nice, but i disliked the username and reputation built upon it. therefore, i created a new one which became my main, hence the apparent confusion.
